### ISSUE TITLE
Unable to learn pixelwise prediction

### ISSUE THREAD

**Issue by shiretzet** [fchollet/keras#2847]
Hi,

First I would like to share my implementation of PixelRNN by Google that I've made with Keras.
https://github.com/shiretzet/PixelRNN

My problem is that the network learns to predict the same value for all pixels. I thought it might be a problem with the loss function implementation or with Keras loss handling, because without the learning phase it does produce some result.
(The loss data is handled as 2D and so is the Softmax layer)

Thanks

**Comment by liu6381810** [comment]
Hi @shiretzet  do you solve this problem?
